client/templates/minutes/minutesEditParticipants.js

```diff
@@ -84,6 +84,14 @@ Template.minutesEditParticipants.helpers({
         }
     },
 
+    switch2MultiColumn() {
```

Comment:
This function has different return types. Depending on the if-clause it returns a string or void. This is not a good practice. Luckily, blaze can handle this and it does not produce any issues. So I will, however, merge this.

A possible improvement for this function would be:
```
switch2MultiColumn() {
    let aMin = new Minutes(_minutesID);
    return (aMin.participants.length > 7) ? "multicolumn" : "";
}
```
That makes the code some lines shorter and the function does always return the same type. See this as a little note for the future ;-)